cmakebuild.js

```diff
@@ -15,7 +15,7 @@ function runCmake (opts, target, cb) {
 
     if (opts.debug) args.push('--debug')
 
-    if (opts.format) args.push('--', '--format', opts.format)
+    if (opts.format) args = args.concat(opts.format.split(' '))
```

Comment:
Would this be ok or does it always need to start with the name?

Reply:
I think it's ok since it's for `cmake-js` only.